tests/test_config_flow.py

```
@@ -325,60 +324,96 @@ def test_validate_brackets(display_options, expected):
-        expected (bool): The expected result of the validation.
-
-    """
-    errors = {}
-    result = _validate_known_options(display_options, errors)
-    assert result is expected
-
-
 @pytest.mark.asyncio
 @pytest.mark.parametrize(
-    "display_options,has_error",
+    "display_options,expected_errors",
```

Ruff (0.15.12)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)